src/stores/settings.js

```diff
@@ -13,15 +13,29 @@ export const useSettingsStore = defineStore("settings", {
         useWinpcap: false,
         server: "steam",
       },
+      uploads: {
+        uploadLogs: false,
+        uploadKey: "",
+        apiUrl: process.env.DEBUGGING ? "http://localhost" : "https://api.dps.arsha.io",
```

Comment:
Please add an environment variable for this as you don't have to run the web interface to develop LOA Details itself.

Reply:
Added to [quasar config ](https://github.com/karaeren/loa-details/pull/13/commits/5c38d2b560a07d75d78fd9401fcbd90d1592ac90#diff-8b8d9829edb0b54005392ad1afe4754faa79c151b7a1fbf4c6907c391bcd00da)